### ISSUE TITLE
[Bug]: [bulk insert v2]the values of some deprecated fields should be set to None instead of using an incorrect value

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#31448]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master/2.4
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
<Bulk insert state:
     - taskID          : 448509402707119188,
     - state           : Started,
     - row_count       : 0,
     - infos           : {'failed_reason': '', 'progress_percent': '50'},
     - id_ranges       : [],
     - create_ts       : 1970-01-01 00:00:00
 >}
```

the value of row_count and create_ts should be set to none instead of a wrong value

**Comment by bigsheeper** [comment]
/assign @zhuwenxing 
please help to make a verify

**Comment by zhuwenxing** [comment]
verified and fixed in `2.4-20240401-4f3bfe24a-amd64`
```
[{'task_id': '448779842399488307', 'state': {'collectionName': 'bulk_insert_5kY81p8t', 'completeTime': '2024-04-01T08:42:49Z', 'details': [{'completeTime': '2024-04-01T08:42:47Z', 'fileName': 'id:448779842399488308 paths:"data-fields-9-rows-4710000-dim-128-file-num-1-error-none-1711960137.parquet" ', 'fileSize': 14646604689, 'importedRows': 4710000, 'progress': 100, 'state': 'Completed', 'totalRows': 4710000}], 'fileSize': 14646604689, 'importedRows': 4710000, 'jobId': '448779842399488307', 'progress': 100, 'state': 'Completed', 'totalRows': 4710000}}]
```
